src/avatar.cpp

```diff
@@ -1700,8 +1701,9 @@ std::string avatar::total_daily_calories_string() const
         // 5 for the day and the offset from it,
         // 18 for the numbers, and 9 for the spacing between them
         // For the second, 4 offset + 5 labels + 8 spacing leaves 15 for the levels
-        std::string activity_str = string_format( "%3dE  %3dA  %3dM  %3dL  %3dN",
+        std::string activity_str = string_format( "%3dE  %3dA  %3dB  %3dM  %3dL  %3dN",
```

Comment:
My fancy formatting! Could I suggest dropping the the length specifiers so this (probably) won't poke out? And update the comment above.